managed/services/telemetry/config.default.yml

```diff
@@ -92,14 +92,12 @@ telemetry:
     summary: "Plugins on the MySQL instance"
     data:
       - metric_name: "1"
-        label: "library"
-      - metric_name: "2"
```

Comment:
why library was removed?

Reply:
if response is MySQL version based then shouldn't it be more complex query?

Reply:
> why library was removed?

`library` is no longer available from the metrics.

Reply:
My bad )

I realized there is no harm if the config has a metric that can potentially be absent - it will still handle the payload without faltering.

I put it bask and added a test.